Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py

```diff
@@ -761,6 +776,15 @@ def Go_To_Link(step_data, page_title=False):
             elif left == "driverid":
                 driver_id = right.strip()
 
+            # checks for capabilities and modifies them by the given step_data
+            elif mid.strip().lower() == "capability":
+                if left.strip().lower() in ("promptbehavior", "alertbehavior"):
+                    if right.strip().lower() in ("accept", "yes", "ok"):
```

Comment:
@Muntasib-creator @Oytizzo there should be another `else` branch to include whatever desired capability the user may provide in the future. The key will be the left and the value will be the right. Can go into next release.